tailwind.config.js

```diff
@@ -4,7 +4,227 @@ module.exports = {
+        navbarDropdownAnimationReverse: {
+          "0%": {
+            transform: "translateY(0%)"
+          },
+          "25%": {
+            transform: "translateY(-25%)"
+          },
+          "50%": {
+            transform: "translateY(-50%)"
+          },
+          "75%": {
+            transform: "translateY(-75%)"
+          },
+          "100%": {
+            transform: "translateY(-100%)"
+          }
+        },
+        navbarMobileDropdownAnimation: {
+          "0%": {
+            transform: "translateX(-100%)"
+          },
+          "25%": {
+            transform: "translateX(-75%)"
+          },
+          "50%": {
+            transform: "translateX(-50%)"
+          },
+          "75%": {
+            transform: "translateX(-25%)"
+          },
+          "100%": {
+            transform: "translateX(0%)"
+          }
+        },
+        navbarMobileColorAnimation: {
+          "0%": {
+            background: "#2E2E38"
+          },
+          "100%": {
+            background: "#3F3FE4" 
+          }
+        },
+        navbarMobileDropdownAnimationReverse: {
+          "0%": {
+            transform: "translateX(0%)"
+          },
+          "25%": {
+            transform: "translateX(-25%)"
+          },
+          "50%": {
+            transform: "translateX(-50%)"
+          },
+          "75%": {
+            transform: "translateX(-75%)"
+          },
+          "100%": {
+            transform: "translateX(-100%)"
+          }
+        },
+        navbarMobileColorAnimationReverse: {
+          "0%": {
+            background: "#3F3FE4"
+          },
+          "100%": {
+            background: "#2E2E38" 
+          }
+        }
+      },
+      animation: {
+        navbarMenuTopAnimation: "navbarMenuTopAnimation .2s ease-in-out forwards",
+        navbarMenuTopAnimationReverse: "navbarMenuTopAnimationReverse .2s ease-in-out forwards",
+        navbarMenuBottomAnimation: "navbarMenuBottomAnimation .2s ease-in-out forwards",
+        navbarMenuBottomAnimationReverse: "navbarMenuBottomAnimationReverse .2s ease-in-out forwards",
+        navbarDropdownAnimation: "navbarDropdownAnimation .2s ease-in-out forwards ",
+        navbarDropdownAnimationReverse: "navbarDropdownAnimationReverse .2s ease-in-out forwards",
+        navbarMobileDropdownAnimation: "navbarMobileDropdownAnimation .2s ease-in-out forwards",
+        navbarMobileColorAnimation: "navbarMobileColorAnimation .2s ease-in-out forwards",
+        navbarMobileDropdownAnimationReverse: "navbarMobileDropdownAnimationReverse .2s ease-in-out forwards",
+        navbarMobileColorAnimationReverse: "navbarMobileColorAnimationReverse .2s ease-in-out forwards"
+      },
+      height: {
+        "navbar-full-height": "70px" 
+      },
+      zIndex: {
+        "negative": "-1",
+        "10": "10"
+      },
+      borderRadius: {
+        "input-radius": "3px"
+      },
+      padding: {
+        "input-padding": "0px 12px 0px 12px"
+      }
```

Comment:
These are more like `utilities`, I thinks it's better you define these in global styling, example: 
```
@layer utilities {
  .input-radius: {
    @apply rounded-[3px]
  }
}
```
and then just use it as `className="input-radius"`